### ISSUE TITLE
M1 macs?

### ISSUE THREAD

**Issue by Rod-Tav** [francoisferland/casiousbmididriver#44]
Hi,
I installed the driver on my m1 MacBook pro, and nothing happens. It worked on the same software on my previous laptop. Is it just an Intel/M1 issue?

**Comment by francoisferland** [comment]
Hi,

I have not had access to a M1 Mac yet, so I was not able to prepare a driver for them. However, someone made a new version (based on this one) that might work, but you will have to compile yourself : https://github.com/par0-git/casiousbmididriver

I will try providing a new universal and installable version as soon as I can test it properly.

**Comment by francoisferland** [comment]
Hello again,

I had time to recompile the driver this afternoon. Obviously still cannot test it on a M1, but could you give me an update ? I can confirm it at least work on Big Sur.

**Comment by Rod-Tav** [comment]
Hi,
Thank you for compiling and posting the download! I was able to test on an M1 Macbook Pro with Big Sur and an M1 Macbook Air with Monterey, and both worked!
When downloading, Chrome flags it as "not commonly downloaded and may be dangerous", so after saying Keep it downloads. Then when opening Garageband, Apple came up with a message saying unable to open so I allowed it in system preferences, and then when opening Garageband for the second time Apple's message asked one last time if I wanted to open the plugin and I hit Allow, and it seems to be good now. I don't know if there's a way around these security checks, but either way it works!
I tested it on Garageband on Big Sur and both Garageband and Logic Pro X on Moterey.

Thanks again!!
